apps/masterbots.ai/components/routes/profile/user-card.tsx

```
@@ -1,17 +1,20 @@
-import { Button } from '@/components/ui/button'
 import { Separator } from '@/components/ui/separator'
-import { UserPersonalityPrompt } from '@/lib/constants/prompts'
-import { useUploadImagesCloudinary } from '@/lib/hooks/use-cloudinary-upload'
-import { useModel } from '@/lib/hooks/use-model'
+import Image from 'next/image'
+import { BookUser, BotIcon, MessageSquareHeart, Wand2, ImagePlus, Loader, UserIcon, Users } from 'lucide-react'
+import { Button } from '@/components/ui/button'
+import { User } from 'mb-genql'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)